modules/sdk-coin-bera/src/lib/walletUtil.ts

```diff
@@ -0,0 +1,604 @@
91506130d98261309a565b602082019050919050565b5f6020820190508181035f8301526130fb816130c2565b9050919050565b7f5369676e6572732063616e6e6f7420626520657175616c0000000000000000005f82015250565b5f613136601783612679565b915061314182613102565b602082019050919050565b5f6020820190508181035f8301526131638161312a565b9050919050565b5f81519050919050565b5f61317e8261316a565b61318881856128ee565b9350613198818560208601612705565b80840191505092915050565b5f6131af8284613174565b915081905092915050565b5f815190506131c881612314565b92915050565b5f602082840312156131e3576131e2611cb4565b5b5f6131f0848285016131ba565b91505092915050565b7f5472616e7366657248656c7065723a3a736166655472616e736665723a2074725f8201527f616e73666572206661696c656400000000000000000000000000000000000000602082015250565b5f613253602d83612679565b915061325e826131f9565b604082019050919050565b5f6020820190508181035f83015261328081613247565b9050919050565b7f496e73756666696369656e742066756e647300000000000000000000000000005f82015250565b5f6132bb601283612679565b91506132c682613287565b602082019050919050565b5f6020820190508181035f8301526132e8816132af565b9050919050565b5f6132fa5f836128ee565b915061330582611bc6565b5f82019050919050565b5f613319826132ef565b9150819050919050565b7f43616c6c206661696c65640000000000000000000000000000000000000000005f82015250565b5f613357600b83612679565b915061336282613323565b602082019050919050565b5f6020820190508181035f8301526133848161334b565b9050919050565b5f60608201905061339e5f830186611b8f565b6133ab6020830185611b8f565b6133b86040830184611ba7565b949350505050565b7f496e76616c6964207369676e6174757265202d2077726f6e67206c656e6774685f82015250565b5f6133f4602083612679565b91506133ff826133c0565b602082019050919050565b5f6020820190508181035f830152613421816133e8565b9050919050565b5f61343282612ffe565b915061343d83612ffe565b9250828201905060ff81111561345657613455612ad9565b5b92915050565b7f45434453413a20696e76616c6964207369676e6174757265202773272076616c5f8201527f7565000000000000000000000000000000000000000000000000000000000000602082015250565b5f6134b6602283612679565b91506134c18261345c565b604082019050919050565b5f6020820190508181035f8301526134e3816134aa565b9050919050565b6134f381612ffe565b82525050565b5f60808201905061350c5f830187612a0e565b61351960208301866134ea565b6135266040830185612a0e565b6135336060830184612a0e565b95945050505050565b7f53657175656e636520494420616c7265616479207573656400000000000000005f82015250565b5f613570601883612679565b915061357b8261353c565b602082019050919050565b5f6020820190508181035f83015261359d81613564565b9050919050565b7f53657175656e63652049442062656c6f772077696e646f7700000000000000005f82015250565b5f6135d8601883612679565b91506135e3826135a4565b602082019050919050565b5f6020820190508181035f830152613605816135cc565b9050919050565b7f53657175656e63652049442061626f7665206d6178696d756d000000000000005f82015250565b5f613640601983612679565b915061364b8261360c565b602082019050919050565b5f6020820190508181035f83015261366d81613634565b905091905056fea26469706673582212200c5a1256669073f019b8de41e297f5970cf2463edf5b2a09c205aec4be9c2dc964736f6c63430008140033';
```

Comment:
I think this file is mainly needed for v0 contracts, you can remove this file

Reply:
i am still able to see this file

Reply:
done